ansible/roles/adm/tasks/main.yml

```diff
@@ -4,6 +4,10 @@
 # point of truth for the keys, unknown keys are dropped, that's why
 # `authorized_key` ansible module is not used.
 
+- name: Run "apt-get update"
+  apt:
+    update_cache: yes
+
```

Comment:
I guess this is the place for a more general question for @darkk and @hellais: how do we manage updates on the remote machines? My limited understanding is that, as of now, we just `apt-get update` when we deploy new stuff, is that correct? Do we turn on `unattended-upgrades` or something similar for _security_ updates?